nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/test/comment.test.js

```diff
@@ -59,7 +63,7 @@ describe("Comments for Sequelize", () => {
     });
 
     after(() => {
-        tracing.stop();
+        opencensus_tracing.stop();
```

Comment:
We don't need to manually test opencensus_tracing any more given that we'll be abstracting it into a package that can be multiplexed on by name.

Reply:
This was to rename "tracing" to make it more obvious which library it was interacting with. The previous tracing test is still fully functional and passing, so I wasn't sure about removing them.